db/blob/db_blob_index_test.cc

```diff
@@ -171,8 +174,13 @@ TEST_F(DBBlobIndexTest, Get) {
     ASSERT_EQ("value", GetImpl("key", &is_blob_index));
     ASSERT_FALSE(is_blob_index);
     // Verify blob index
-    ASSERT_TRUE(Get("blob_key", &value).IsNotSupported());
-    ASSERT_EQ("NOT_SUPPORTED", GetImpl("blob_key"));
+    if (tier <= kImmutableMemtables) {
```

Comment:
Should the comment on line 159 be updated?

Reply:
Yes, nice catch, thanks!